contrib/epee/include/net/enums.h

```diff
@@ -42,7 +42,8 @@ namespace net_utils
 		ipv4 = 1,
 		ipv6 = 2,
 		i2p = 3,
-		tor = 4
+		tor = 4,
+		ipv4subnet = 5,
```

Comment:
This seems like an abuse of the interface - its no longer representing a single address endpoint and is now representing a range of inputs. In the majority of contexts where `network_address` is being used, this new variant will not be wanted.

Reply:
So what do you suggest instead ?